policy/engine_test.go

```diff
@@ -2,17 +2,37 @@ package policy
 
 import (
 	"context"
+	"io/ioutil"
+	"os"
 	"testing"
 
 	"github.com/open-policy-agent/conftest/parser"
 	"github.com/open-policy-agent/opa/ast"
 )
 
+func TestCapabilities(t *testing.T) {
```

Comment:
Unit tests shouldn't perform disk I/O. Consider using an `io.Reader` (or similar) type in the signature for the `Load()` function rather than a string for the path to read the JSON from disk. This would enable you to test without disk I/O.

Reply:
Indeed... I've just moved the determination of the value of capabilities to upper context(LoadWithData), This resulted the test being unnecessary as we have no test that's covering `LoadWithData()` 
 but if we prefer a having test for json works (os.open, load etc) to increase the coverage, I can export it to a function and test it using `testing/fstest` or some else way
Thoughts? Would it worth it or having a comprehensive e2e is enough for this case?

Reply:
Still I can test LoadWithData() based on existing file on `examples/` Given we will have one example for capabilities, we may prefer having something that'd look like:
```go
	ctx := context.Background()
	policyPath := []string{"../examples/exceptions/policy"}
	capabilitiesPath := "../examples/capabilities/capabilities.json"
        
        // set up table test structure
	engine, err := LoadWithData(ctx, policyPath, []string{""}, capabilitiesPath)
        // check err
```

Reply:
I would like LoadWithData() to have its own unit tests, but that would require some refactoring. Since the new capabilities are covered in the acceptance tests, I think this is OK.